server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -250,7 +251,7 @@ static class FateOpsCommand {
     boolean printJson;
 
     @Parameter(names = {"-s", "--state"},
-        description = "<state>[ -s <state>...] Print transactions in the state(s) {NEW, IN_PROGRESS, FAILED_IN_PROGRESS, FAILED, SUCCESSFUL}")
+        description = "<state>... Print transactions in the state(s) {NEW, IN_PROGRESS, FAILED_IN_PROGRESS, FAILED, SUCCESSFUL}")
```

Comment:
I still don't think we need this filtering. Users can filter by piping the output to grep, or whatever they like. We can put the list of states elsewhere in the help/description. The output should be easily computer-readable, to make parsing the output easy... but otherwise, we don't need to bake in too many filtering/display options. Let's keep it simple.

Reply:
Except that its just carrying over functionality from when it was a shell command - but either way....

Reply:
Okay. I don't know that we need to be tied to that same flag in the new tool, as long as we preserve all the same functionality, and filtering functionality is still easy and convenient, regardless of whether we carry over this flag. I'm not a big fan of carrying it over, but I'm not strongly opposed either... I just think it's unnecessary and redundant, and I'd prefer to delete it.

Reply:
Be sure to check FateSummaryIT

Reply:
Thanks for the reminder. FateSummaryIT is passing.

The functionality of the filtering is pretty simple and just calls the same methods as FateCommand. It is only a few lines of code to make the option available in the new command.